layouts/funding_calls/list.html

```diff
@@ -40,154 +37,68 @@ <h4>Potential funding sources</h4>
-                          </div>
-                          <div class="modal-body">
-                            <div class="row mt-1">
-                              <div class="col">
-                                <b>Application deadline:</b> ${item.deadline_time ? long_date + " - " + item.deadline_time : long_date}
-                              </div>
-                            </div>`;
-
-                            if(funder){ 
-                            blockHTML += `
-                            <div class="row mt-2">
-                              <div class="col">
-                                <b>Funder:</b> ${ item.funder }
-                              </div>
-                            </div>`;
-                            } 
-
-                            if(item.description) {
-                              blockHTML += `
-                              <div class="row mt-2">
-                                <div class="col"><b>Brief description:</b><br>
-                                  ${item.description}
-                                </div>
-                              </div>`;
-                            }
-
-                            if(item.applicant){
-                              blockHTML += `
-                              <div class="row mt-2">
-                                <div class="col"><b>Who can apply?</b><br>
-                                  ${item.applicant}
-                                </div>
-                              </div>`;
-                            }
-                            
-
-                            blockHTML += `<div class="row mt-3">
-                              <p>For more information, visit the link below.</p>
-                              <div class="col-md">
-                                <a target="_blank" href="${item.url}"><i class="bi bi-globe"></i> Call webpage</a>
-                              </div>
-                            </div>
-                          </div>
-                          <div class="modal-footer">
-                            <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Close</button>
-                          </div>
-                        </div>
-                      </div>
-                    </div>
-                  </div>
-                </div>`;
-
-            blockHTML += `
+  // Takes a URL and fetchs all open funding calls with label "Data Centre"
+  async function getDCFundingCalls(url, label="Data Centre") {
+    try {
+      // Fetch DC funding directly from Anubis using API
+      const response = await fetch(url);
+      const allData = await response.json();
+      const allFund = allData.calls.filter((f) => f.labels.includes(label))
+      return allFund.sort((a,b) => (a.closes > b.closes) ? 1 : (a.closes < b.closes) ? -1 : 0);
+    } catch (error) {
+      console.error('Error fetching events:', error);
```

Comment:
minor typo. Here console message should say fundings.
```suggestion
      console.error('Error fetching fundings:', error);
```